docs/api/osquery-manager/live-queries/create.asciidoc

```diff
@@ -49,6 +49,8 @@ experimental[] Create live queries.
 
 `metadata`:: (Optional, object) Custom metadata object associated to the live query.
 
+`timeout`:: (Optional, number) A timeout period, in seconds, after which the query will stop running. Overwriting the default timeout allows you to support queries that require more time to complete. The default and minimum supported value is `60`. The maximum supported value is `900`.
```

Comment:
@szwarckonrad @tomsonpl just double-checking that the data type for `timeout` is number, not integer?

Reply:
Not sure, could you explain the difference between number and integer in this case?